CHANGELOG.md

```diff
@@ -3,16 +3,25 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Version 1.0 - [2023-02-15]
-
-Initial release of nf-core/phyloplace, created with the [nf-core](https://nf-co.re/) template.
-
-The pipeline performs phylogenetic placement of nucleotide or amino acid sequences in a reference phylogeny.
+## v2.0.0 - [2025-02-13]
```

Comment:
don't see any breaking changes, so what makes this a 2.0 instead of 1.1 release?

Reply:
There's no `--input` any more, but two different for the two modes, so, in my view, a contract change.